examples/timeout/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "jazzerjs-timeout-example",
+	"version": "1.0.0",
+	"description": "An example showing how Jazzer.js handles timeouts",
+	"scripts": {
+		"fuzz": "jazzer fuzz -- -timeout=1",
+		"dryRun": "echo \"skipped\""
```

Comment:
Don't we want to run the example as part of the tests? Is that because the test harness currently can't handle crashing examples?

Reply:
Exactly. We first have to build a way to properly assert fuzzer findings.